sys/kern/sys_process.c

```diff
@@ -742,14 +745,33 @@ sys_ptrace(struct thread *td, struct ptrace_args *uap)
 		error = COPYIN(uap->addr, &r.pve, sizeof r.pve);
 #endif
 		break;
+	case PT_LWPINFO:
+		data = sizeof(r.pl);
```

Comment:
This breaks the wrap32/wrap64 checks for the later PT_LWPINFO case after kern_ptrace returns.

Reply:
Right. I think it is now fixed. Let me know!